blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -274,9 +274,13 @@ public void parallelStart(@Nonnull FlowNode parallelStartNode, @Nonnull FlowNode
                     status = new NodeRunStatus(genericStatus);
                 }
             }else{
-                times = new TimingInfo(TimingAction.getStartTime(branchStartNode)+System.currentTimeMillis(),
+                long startTime = System.currentTimeMillis();
```

Comment:
A little more detail - https://github.com/jenkinsci/workflow-job-plugin/blob/master/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java#L895 gets called from https://github.com/jenkinsci/workflow-cps-plugin/blob/92019ea288775c5d2e357f701472b52e40a7cc99/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThreadGroup.java#L383, right after the node gets created. So the `TimingAction` should only be null during that tiny window between node creation and the action being added. I don't see any other way it could be null.